src/main/application/BorosTcf.js

```diff
@@ -0,0 +1,38 @@
+import {inject} from '../core/ioc/ioc'
+
+import {GetVendorListUseCase} from './vendor_list/GetVendorListUseCase'
+import {GetConsentStatusUseCase} from './vendor_consent/GetConsentStatusUseCase'
+import {LoadUserConsentUseCase} from './vendor_consent/LoadUserConsentUseCase'
+import {SaveUserConsentUseCase} from './vendor_consent/SaveUserConsentUseCase'
```

Comment:
the directory vendor_list, shouldn't be service?? Is a folder for all use cases, and also, all that 4 use cases are not from vendorList domain. So I guess we should change the name of the folder.

Reply:
Adding services folder. done! But, I have 1 use case for vendor_list and 3 use cases for vendor_consent. Is not ok?

Reply:
I don't know if it's necessary to nest that folders, or all the use cases can be together. In the case that you need to add this veendorlist and vendorconsent folders, wouldn't be better to write them as vendorList and vendorConsent?

Reply:
I have followed the same naming that was previously in Boros-cmp. 

![Screenshot 2020-05-07 at 16 37 57](https://user-images.githubusercontent.com/537656/81307736-4752a180-9081-11ea-9d17-48671ca9e7bf.png)

But we can change them.